src/configure.js

```diff
@@ -0,0 +1,16 @@
+let defaults = {
+  adapter    : require('./adapters/ajax'),
+  body       : undefined,
+  params     : undefined,
+  headers    : {},
+  method     : 'GET',
```

Comment:
a lot of this configuration doesn't make sense out of the context of the adapter. should there be a key under which all adapter configuration resides?